applications/zigbee_weather_station/sample.yaml

```diff
@@ -5,6 +5,14 @@ tests:
   applications.zigbee_weather_station:
     build_only: true
     platform_allow: thingy53_nrf5340_cpuapp
+    tags: ci_build
+    extra_args: CONF_FILE=prj_release.conf
     integration_platforms:
       - thingy53_nrf5340_cpuapp
-    tags: ci_build
+  applications.zigbee_weather_station.debug:
+    build_only: true
+    platform_allow: thingy53_nrf5340_cpuapp
+    tags: ci_build debug
+    extra_args: CONF_FILE=prj.conf
+    integration_platforms:
+      -
```

Comment:
Empty integration_platforms?

Reply:
Do you mean by "release version  is enough" some plan which verifies the application before releases? Integration_platforms serves for sdk-nrf CI where at the entry level samples/apps/tests are build with twister. Hexes can be reused in downstream jobs as well. If you don't use integration_platforms here it means this particular CI step will never verify your app (there is only a single scope defined by integration_platforms). 
If you don't want sdk-nrf CI building this app at twister stage just delete integration_platforms entry. In the current form it will cause errors in twister after the ongoing upmerge. Later on I might ask you to add something like `integration_platforms:  - null` but we need to resolve such entry first in the upstream